corehq/pillows/application.py

```diff
@@ -33,13 +33,14 @@ class AppPillow(AliasedElasticPillow):
     es_index = APP_INDEX
     default_mapping = APP_MAPPING
 
+    @classmethod
     @memoized
```

Comment:
Is it OK that these classmethods are memoized. Judging by my initial look it seems like it is but just mentioning anyway.